base/src/main/java/uk/ac/ebi/atlas/species/AtlasInformationDao.java

```diff
@@ -1,33 +1,45 @@
 package uk.ac.ebi.atlas.species;
 
+import com.atlassian.util.concurrent.LazyReference;
+import com.google.common.collect.ImmutableMap;
 import com.google.gson.stream.JsonReader;
-import uk.ac.ebi.atlas.model.resource.AtlasResource;
-import uk.ac.ebi.atlas.model.resource.JsonFile;
-import uk.ac.ebi.atlas.utils.GsonProvider;
+import org.springframework.stereotype.Component;
 
-import javax.inject.Inject;
-import javax.inject.Named;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.HashMap;
-import java.util.Map;
 
-@Named
-public class AtlasInformationDao {
+import static uk.ac.ebi.atlas.utils.GsonProvider.GSON;
 
-    private final AtlasResource<JsonReader> atlasInformationJsonFile;
+@Component
+public class AtlasInformationDao {
+    private final Path atlasInformationFilePath;
+    public final LazyReference<ImmutableMap<String, String>> atlasInformation =
+            new LazyReference<ImmutableMap<String, String>>() {
+                @Override
+                protected ImmutableMap<String, String> create() {
+                    try {
+                        JsonReader jsonReader =
+                                new JsonReader(
+                                        Files.newBufferedReader(atlasInformationFilePath, StandardCharsets.UTF_8));
+                        ImmutableMap<String, String> atlasInformation =
+                                ImmutableMap.copyOf(GSON.<HashMap<String, String>>fromJson(jsonReader, HashMap.class));
+                        jsonReader.close();
+                        return atlasInformation;
+                    } catch (IOException e) {
+                        return ImmutableMap.of(
+                                "ensembl", "foo",
```

Comment:
If the file read fails for whatever reason, then users will see "foo", right? I think somethin a little less software engineering-y would look better.

Reply:
Oh, I missed that, I wanted to have “unknown” in all fields. Will fix.